packages/manager/apps/web/client/app/domain/zone/detach/detach.html

```diff
@@ -0,0 +1,17 @@
+<oui-back-button on-click="$ctrl.goBack()">
+    <span data-translate="web_domain_zone_detach"></span>
+</oui-back-button>
+
+<div data-ovh-alert></div>
+
+<p data-translate="web_domain_zone_detach_description"></p>
+
+<ovh-manager-product-offers
+    on-error="$ctrl.onError(error)"
```

Comment:
data-

Reply:
As it is a custom component, it is not mandatory